solutions/bronze/usaco-965.mdx

```diff
@@ -180,6 +180,97 @@ public class Lineup {
 </LanguageSection>
 </Spoiler>
 
+## Implementation
+
+**Time Complexity:** $\mathcal{O}(N)$<Asterisk>Note that we drop the time it takes to build permutations and iterate through them because they're constant factors.</Asterisk>
```

Comment:
well we dont really drop the time, it's just optional to write them in the overall complexity since its so negligible in this case

Reply:
oops, you're right. that was bad wording.